### PR TITLE
documentation changes to support debian style systems without stepping around the package manager.

### PR DESCRIPTION
**Pull request by julialongtin**

### CODE REVIEW

File: doc/Dependencies.md [hunk 1]
```diff
@@ -7,37 +7,33 @@ In addition to the information below, you can also consult the Dockerfiles for A
...
 
 ```bash
 sudo dnf install -y pkgconfig haskell-platform libstdc++-devel libstdc++-static gcc-c++ libtool automake openssl-devel libsodium-devel ncurses-compat-libs libicu-devel GeoIP-devel libxml2-devel snappy-devel protobuf-compiler
 ```
 
-#### Debian:
+### Ubuntu / Debian:
 
-*Note: Debian is not recommended due to this issue when running local integration tests: [#327](https://github.com/wireapp/wire-server/issues/327)*
+_Note_: Debian is not recommended due to this issue when running local integration tests: [#327](https://github.com/wireapp/wire-server/issues/327)*. This issue does not occur with ubuntu.
```
Comment (neongreen) [inline doc/Dependencies.md]:
```suggestion
_Note_: Debian is not recommended due to this issue when running local integration tests: [#327](https://github.com/wireapp/wire-server/issues/327)*. This issue does not occur with Ubuntu.
```

File: doc/Dependencies.md [hunk 2]
```diff
@@ -7,37 +7,33 @@ In addition to the information below, you can also consult the Dockerfiles for A
...
 
 ```bash
-sudo apt install pkg-config libsodium-dev openssl-dev libtool automake build-essential libicu-dev libsnappy-dev libgeoip-dev protobuf-compiler libxml2-dev -y
+sudo apt install pkg-config libsodium-dev openssl-dev libtool automake build-essential libicu-dev libsnappy-dev libgeoip-dev protobuf-compiler libxml2-dev zlib1g-dev-y
```
Comment (neongreen) [inline doc/Dependencies.md]:
```suggestion
sudo apt install pkg-config libsodium-dev openssl-dev libtool automake build-essential libicu-dev libsnappy-dev libgeoip-dev protobuf-compiler libxml2-dev zlib1g-dev -y
```

File: doc/Dependencies.md [hunk 3]
```diff
@@ -55,31 +51,68 @@ extra-lib-dirs:
...
+When you're done, ensure `stack --version` is >= 1.6.5
+
+You may wish to make executables installed by stack available, by e.g. adding the following to your shell profile:
+
+```bash
+export PATH=~/.local/bin:$PATH
+```
+
+### Ubuntu / Debian Unstable
+_Note_: Debian stretch packages too old of a version of haskell-stack. It is recommended to retrieve the version available from testing, or unstable, or to use stack to update stack.(https://github.com/commercialhaskell/stack/issues/3686)*
```
Comment (fisx) [inline doc/Dependencies.md]: i'm actually using this source, which works well with stretch:

```
https://github.com/commercialhaskell/stack/issues/3686
```

Comment (fisx) [reply doc/Dependencies.md]: > i'm actually using this source, which works well with stretch:
> 
> ```
> deb https://download.fpcomplete.com/debian jessie main
> ```

sorry, cut&paste issue.

**Pull request merged**